_posts/2025-07-09-adbc-19-release.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "Apache Arrow ADBC 19 (Libraries) Release"
+date: "2025-07-09 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the version 19 release of
+the Apache Arrow ADBC libraries. This release includes [**60
+resolved issues**][1] from [**27 distinct contributors**][2].
+
+This is a release of the **libraries**, which are at version 19.  The
+[**API specification**][specification] is versioned separately and is at
+version 1.1.0.
+
+The subcomponents are versioned independently:
+
+- C/C++/GLib/Go/Python/Ruby: 1.7.0
+- C#: 0.19.0
+- Java: 0.19.0
+- R: 0.19.0
+- Rust: 0.19.0
+
+The release notes below are not exhaustive and only expose selected
+highlights of the release. Many other bugfixes and improvements have
+been made: we refer you to the [complete changelog][3].
+
+## Release Highlights
+
+- Apache, Databricks: these drivers has received a plethora of improvements,
```

Comment:
Apache Spark?

```suggestion
- Apache Spark, Databricks: these drivers has received a plethora of improvements,
```

Reply:
The [driver](https://github.com/apache/arrow-adbc/tree/main/csharp/src/Drivers/Apache) is a bit weird and is really for Hive/Impala/Spark. Maybe "Apache Hive/Impala/Spark, Databricks" then?

Reply:
```suggestion
- Apache Hive/Impala/Spark, Databricks: these drivers have received a plethora of improvements,
```